src/i18n/resources/ar/other.json

```diff
@@ -1,8 +1,4 @@
 {
-  "fileDrop": {
-    "dropFilesHere": "ضع الملف هنا",
-    "dragAndDropOrClick": "اسحب و ضع الملفات هنا, او اضغط لاختيار الملف "
-  },
   "anchors": {
     "fra": {
       "2025": {
```

Comment:
As a note, we have a special case here. This "anchors" object is only defined for Arabic. Crowdin only exports strings that are present in the source language file, all other properties are cleaned up. Since `en/other.json` doesn't have this property, Crowdin would delete it. To avoid that, I will extract "anchors" into its own file for arabic only. 

This is an example from the dummy project I created to test:
![image](https://github.com/user-attachments/assets/741a8785-4f13-46ba-95a7-b14d731590b6)